### PR TITLE
Pypi

### PR DESCRIPTION
**Pull request by MaxNoe**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[metadata]
+name = proposal
+url = https://github.com/tudo-astroparticlephysics/PROPOSAL
+author = PROPOSAL Developement Team
+author-email = jan.soedingrekso@tu-dortmund.de
+classifiers=
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Programming Language :: Python :: 3
```
Comment (sudojan) [inline setup.cfg]: python 2?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Physics
+    Development Status :: 5 - Production/Stable
+
+description = Template tool for putting up the scaffold of a Python project
```
Comment (sudojan) [inline setup.cfg]: copy-paste?
Monte-Carlo Simulation library to propagate leptons

File: setup.cfg (hunk 1 above)
Comment (MaxNoe) [reply setup.cfg]: Python 2 had end of life 2020-01-01

File: setup.cfg (hunk 2 above)
Comment (MaxNoe) [reply setup.cfg]: Leptons? Or Leptons and Gamma rays?

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Physics
+    Development Status :: 5 - Production/Stable
+
+license = GPLv3
+description = Monte Carlo simulation library to propagate leptons and gamma rays
```
Comment (sudojan) [inline setup.cfg]: high energy photons or gamma photons
gamma rays sound like it can propagate through space including EBL absorption

Comment (MaxNoe) [reply setup.cfg]: That's all just names for the same thing. maybe addinge "dense media" ?

**Pull request merged**